### PR TITLE
Files - get trashed file

### PR DESCRIPTION
**Pull request by Trudko**
Implementation, documentation and unit tests.

### CODE REVIEW

File: docs/files.md [hunk 1]
```diff
@@ -113,6 +114,14 @@ method will move the file to the user's trash.
 ```js
 client.files.delete('12345', callback);
 ```
+Get a Trashed File
+---------------
+
+Information about a file in the trash can be retrieved with the [`files.getTrashedFile(fileID, callback)`](http://opensource.box.com/box-node-sdk/Folders.html#getTrashedFile) method.
```
Comment (mattwiller) [inline docs/files.md]: The URL to the docs should be http://opensource.box.com/box-node-sdk/Files.html#getTrashedFile

File: lib/managers/files.js [hunk 2]
```diff
@@ -560,6 +560,22 @@ Files.prototype.deleteMetadata = function(fileID, scope, template, callback) {
...
+ * Retrieves a file that has been moved to the trash.
+ *
+ * API Endpoint: '/files/:fileID/trash'
+ * Method: GET
+ *
+ * @param {string} fileID - The ID of the file being requested
+ * @param {Function} callback - Called with nothing if successful, error otherwise
+ * @returns {void}
+ */
+Files.prototype.getTrashedFile = function(fileID, callback) {
```
Comment (mattwiller) [inline lib/managers/files.js]: I think this API endpoint accepts the fields parameter, so we probably need to include the ability to pass qs here.

Comment (Trudko) [inline lib/managers/files.js]: [Does it?](https://docs.box.com/reference#get-a-trashed-file)

Comment (mattwiller) [inline lib/managers/files.js]: Yes, it does:

```
12:09 $ curl https://api.box.com/2.0/files/68475254229/trash?fields=id,name -H "Authorization: Bearer ACCESS_TOKEN"
{"type":"file","id":"68475254229","etag":"2","name":"My Document.pdf"}
```

In general, almost all GET requests accept the fields parameter, even if it isn't explicitly mentioned.

Comment (Trudko) [inline lib/managers/files.js]: Added qs parameter and documentation similar to get function documentation.

**Comment by Trudko** [comment]
Conflicts resolved

File: lib/managers/files.js [hunk 3]
```diff
@@ -575,6 +575,27 @@ Files.prototype.deletePermanently = function(fileID, callback) {
...
 
+/*
+ * Retrieves a file that has been moved to the trash.
+ *
+ * API Endpoint: '/files/:fileID/trash'
+ * Method: GET
+ *
+ * @param {string} fileID - The ID of the file being requested
+ * @param {?Object} qs - Additional options can be passed with the request via querystring. Can be left null in most cases.
+ * @param {Function} callback - Called with nothing if successful, error otherwise
```
Comment (mattwiller) [inline lib/managers/files.js]: This comment needs to be updated; the callback is called with information about the file when successful

Comment (Trudko) [inline lib/managers/files.js]: Comment updated.

**Comment by Trudko** [comment]
Conflicts are resolved in this branch, can you merge it or is there something I should do?

**Pull request merged**